cobra/flux_analysis/moma.py

```diff
@@ -9,7 +9,7 @@
 from cobra.solvers import get_solver_name, solver_dict
 
 
-def add_moma(model):
+def add_moma(model, linear=False):
     """Add constraints and objective representing a MOMA
     (minimization of metabolic adjustment) model.
 
```

Comment:
perhaps update the doc string, new parameter, here as well and brief explanation pro/con linear